project/PlayCrossCompilation.scala

```diff
@@ -1,4 +1,11 @@
-import uk.gov.hmrc.playcrosscompilation.AbstractPlayCrossCompilation
+import uk.gov.hmrc.playcrosscompilation.{AbstractPlayCrossCompilation, PlayVersion}
 import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play25
 
-object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play25)
\ No newline at end of file
+object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play25){
+  override def playCrossScalaBuilds(scalaVersions: Seq[String]): Seq[String] =
```

Comment:
if you set the default scalaversion to 2.12 in build.sbt - then you won't need to override playCrossScalaBuilds.